requirements/customer/cli.tex

```diff
@@ -94,6 +94,9 @@ \section{User Client}
 {If set to a string that can be interpreted as RAM amount, informs the server that this amount of RAM is required for the job.}
 \jaoption{threads}{t}
 {If set to an integer, informs the server that this number of threads is required for the job.}
+\jaoption{base}{z}
```

Comment:
How about renaming the option to environment with abbreviation e?

Reply:
I was thinking z is closer to the pronunciation of "base", at least in my mind.

Reply:
Sure, but no one will be able to remember z as an abbreviation for base.
I think it would be better to do one of the following:

- Change the option name so that the abbreviation is not in use.
- Allow upper-case abbreviations.
- Allow two-letter abbreviations.

Reply:
Maybe -i for image?

Reply:
Would also work I think.